v63002/u_inref/gtm8698.csh

```diff
@@ -0,0 +1,34 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+echo "# Changing the region to AUTODB and setting the file to an invalid .dat file"
+echo "# This will trigger an error when we attempt to update the database"
+$GDE change -segment DEFAULT -file_name="a/a.dat" >& changefile.out
+$GDE change -region DEFAULT -AUTODB >& autodb.out
+rm mumps.dat
+set t = `date +"%b %e %H:%M:%S"`
+sleep 1
```

Comment:
This sleep seems unnecessary.

Reply:
I believe the way getoper works is it starts checking the first second after the set time, meaning if the message were to come in the same second, it will not pick it up. As such, I have adopted the practice of always including the sleep 1 after I set the time to avoid this (I did not observe this behavior on this test but it has caused me issues on other tests)

Reply:
As discussed, getoper.csh gets messages from the start of the set time, not 1 second after. So we will remove any sleeps whose need we are not sure of. And reintroduce them (with comments on their need) if we later realize the need.